crates/astria-proto/buf.gen.yaml

```diff
@@ -8,6 +8,8 @@ plugins:
      - extern_path=.tendermint.types=::tendermint-proto::types
      - extern_path=.tendermint.abci=::tendermint-proto::abci
      - type_attribute=.astria.primitive.v1.Uint128=#[derive(Copy)]
+     - type_attribute=.astria.sequencer.v1alpha1.MintAction=#[cfg(feature = "mint")]
+     - field_attribute=.astria.sequencer.v1alpha1.Action.value.mint_action=#[cfg(feature = "mint")]
```

Comment:
I don't know if feature gating this here is necessary or advisable. I can see why, but at the same time I wonder if this is going to lead to strange errors.

Reply:
it won't compile otherwise :( due to the feature gating in sequencer, i also need to do feathre gating here, otherwise will complain due to missing enum match arms